etna/packages/magmaR/R/update.R

```diff
@@ -106,7 +107,7 @@ updateValues <- function(
         num_new <- length(rec_names_new)
         
         ### Summarize for NEW records
-        cat("For model \"", modelName, "\", this update() will create ", num_new, " NEW records:\n    ",
+        cat("For model \"", modelName, "\", this update() will create (or update) ", num_new, " NEW (or orphan) records:\n    ",
```

Comment:
Addresses #431 by adjusting the 'NEW' records section of the update summary to report that they could also be orphan records.